charts/kuadrant-operator/README.md

```diff
@@ -2,59 +2,60 @@
 
 This is the Helm Chart to install the official Kuadrant Kubernetes Operator
 
-## Installation
+## Dependencies
 
-### Prerequisites
+Follow the steps below to install the following dependencies for Kuadrant Operator:
 - [Gateway API](https://gateway-api.sigs.k8s.io/)
 - [cert-manager](https://cert-manager.io/)
-- A Gateway Provider (I.E: [Istio](https://istio.io/latest/docs/ambient/install/helm/),
-  [Envoy](https://www.envoyproxy.io/docs/envoy/latest/start/start))
+- A Gateway Provider [Istio](https://istio.io/latest/docs/ambient/install/helm/)
 
-### Install dependencies
+[Envoy](https://www.envoyproxy.io/docs/envoy/latest/start/start) is also an option as a gateway provider, **Steps Coming soon!**
+
+### Install Dependencies
 
 1. Install Kubernetes Gateway API:
 
-```sh
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
-```
+    ```sh
+    kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+    ```
 
 2. Install cert-manager
 
-```sh
-helm repo add jetstack https://charts.jetstack.io --force-update
-helm install \
-  cert-manager jetstack/cert-manager \
-  --namespace cert-manager \
-  --create-namespace \
-  --version v1.15.3 \
-  --set crds.enabled=true
-```
-
-3. Install a Gateway Controller, i.e: Istio:
-
-```sh
-helm install sail-operator \
-		--create-namespace \
-		--namespace istio-system \
-		--wait \
-		--timeout=300s \
-		https://github.com/istio-ecosystem/sail-operator/releases/download/0.1.0/sail-operator-0.1.0.tgz
-
-kubectl apply -f -<<EOF
-apiVersion: sailoperator.io/v1alpha1
-kind: Istio
-metadata:
-  name: default
-spec:
-  # Supported values for sail-operator v0.1.0 are [v1.22.4,v1.23.0]
-  version: v1.23.0
-  namespace: istio-system
-  # Disable autoscaling to reduce dev resources
-  values:
-    pilot:
-      autoscaleEnabled: false
-EOF
-```
+    ```sh
```

Comment:
Why the 4 space indent here.
Are the backticks not enough to render the code block?

Reply:
it does render as is but the numbered headings are producing an error then. this change isn't essential tho so if you'd prefer to have it removed I can do that.

Reply:
Funny you should mention the numbered headings.
I'm proposing we avoid numbered lists in this [style guide PR](https://github.com/Kuadrant/docs.kuadrant.io/pull/183/files#diff-73ed2b26f52326ccd45aaa81e052bab2f213fb4e521a6f231bbb629d6dfb8b96R95-R97).

Personally, I think that should extend to numbering of headers as well, for the same reason.

re: the spaces on the code blocks,
for consistency would you mind if we removed the indentation, to align with the usage docs in https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#usage